arangod/RocksDBEngine/RocksDBVPackIndex.cpp

```diff
@@ -874,12 +934,24 @@ Result RocksDBVPackIndex::insert(transaction::Methods& trx,
 
   // now we are going to construct the value to insert into rocksdb
   if (_unique) {
-    // unique indexes have a different key structure
+    // build index value (storedValues array will be stored in value if
+    // storedValues are used)
     RocksDBValue value = RocksDBValue::UniqueVPackIndexValue(documentId);
+    if (!_storedValuesPaths.empty()) {
+      transaction::BuilderLeaser leased(&trx);
+      leased->openArray(true);
+      for (auto const& it : _storedValuesPaths) {
+        leased->add(doc.get(it));
```

Comment:
What do we want to in case of missing attribute? For the `fields` we either stop indexing document (sparse index)  or forcefully put `null` slice. As I see here we may put `none`. Maybe always force `null` for consistency with `fields`?

Reply:
Addressed.